src/app/onboarding/legal-credentials/page.tsx

```diff
@@ -45,14 +46,21 @@ export default function Page() {
       stateBarred: onboarding.profile.state_barred ?? undefined,
       barNumber: onboarding.profile.bar_number ?? undefined,
       eoirRegistered: onboarding.profile.eoir_registered ?? undefined,
+      legalCredentialComment:
+        onboarding.profile.legal_credential_comment ?? undefined,
+      barred:
+        onboarding.profile.bar_number === undefined
+          ? undefined
+          : onboarding.profile.bar_number !== 'Not Barred',
```

Comment:
i'll add a separate boolean in supabase to indicate it, since the user can technically input "Not Barred" in the bar number field
and we can add any message depending on the boolean